src/editor/HostDialog.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>424</width>
-    <height>352</height>
```

Comment:
is this resize intentional? the new default seems a bit small to me
![image](https://user-images.githubusercontent.com/9067601/189546309-e257e37d-65ba-4ab5-8eac-049b1805657e.png)

Reply:
The new one is on the right, yeah? Can make it bigger if need be, but it is bigger than the old default iirc

Reply:
The new one's on the left I think. 
![image](https://user-images.githubusercontent.com/9067601/192104606-f063dd71-15bd-4f6f-95cc-0d5328d5385a.png)
I'll revert the change there then.